### PR TITLE
add arcade pinball table packs

### PR DESCRIPTION
**Pull request by Omeric**

### CODE REVIEW

File: classic/com.farsight.AndroidPinball.javaProject.json [hunk 1]
```diff
@@ -1052,7 +1062,7 @@
             "promoted": false,
             "type": "entitlement",
             "identifier": "jackbot_promode",
-            "name": "JackBot Pro Mode",
+            "name": "JackBot Pro Upgrade",
```
Comment (Szeraax) [inline classic/com.farsight.AndroidPinball.javaProject.json]: Keep this change? @Omeric

File: classic/com.farsight.AndroidPinball.javaProject.json [hunk 2]
```diff
@@ -1562,7 +1662,7 @@
             "promoted": false,
             "type": "entitlement",
             "identifier": "season_pack_1_pro",
-            "name": "Season One Pro Pack",
+            "name": "Season One Pack Pro",
```
Comment (Szeraax) [inline classic/com.farsight.AndroidPinball.javaProject.json]: Planning to revert

File: classic/com.farsight.AndroidPinball.javaProject.json [hunk 3]
```diff
@@ -1582,7 +1682,7 @@
             "promoted": false,
             "type": "entitlement",
             "identifier": "season_pack_2_pro",
-            "name": "Season Two Pro Pack",
+            "name": "Season Two Pack Pro",
```
Comment (Szeraax) [inline classic/com.farsight.AndroidPinball.javaProject.json]: Planning to revert

File: classic/com.farsight.AndroidPinball.javaProject.json [hunk 4]
```diff
@@ -1602,7 +1702,7 @@
             "promoted": false,
             "type": "entitlement",
             "identifier": "season_pack_3_pro",
-            "name": "Season Three Pro Pack",
+            "name": "Season Three Pack Pro",
```
Comment (Szeraax) [inline classic/com.farsight.AndroidPinball.javaProject.json]: Planning to revert

File: classic/com.farsight.AndroidPinball.javaProject.json [hunk 5]
```diff
@@ -1622,7 +1722,7 @@
             "promoted": false,
             "type": "entitlement",
             "identifier": "season_pack_4_pro",
-            "name": "Season Four Pro Pack",
+            "name": "Season Four Pack Pro",
```
Comment (Szeraax) [inline classic/com.farsight.AndroidPinball.javaProject.json]: Planning to revert

File: classic/com.farsight.AndroidPinball.javaProject.json [hunk 6]
```diff
@@ -1642,7 +1742,7 @@
             "promoted": false,
             "type": "entitlement",
             "identifier": "season_pack_5_pro",
-            "name": "Season Five Pro Pack",
+            "name": "Season Five Pack Pro",
```
Comment (Szeraax) [inline classic/com.farsight.AndroidPinball.javaProject.json]: Planning to revert

File: classic/com.farsight.AndroidPinball.javaProject.json [hunk 7]
```diff
@@ -1662,7 +1762,7 @@
             "promoted": false,
             "type": "entitlement",
             "identifier": "season_pack_6_pro",
-            "name": "Season Six Pro Pack",
+            "name": "Season Six PackPro",
```
Comment (Szeraax) [inline classic/com.farsight.AndroidPinball.javaProject.json]: Planning to revert

File: classic/com.farsight.AndroidPinball.javaProject.json [hunk 8]
```diff
@@ -1682,7 +1782,7 @@
             "promoted": false,
             "type": "entitlement",
             "identifier": "season_pack_7_pro",
-            "name": "Season Seven Pro Pack",
+            "name": "Season Seven Pack Pro",
```
Comment (Szeraax) [inline classic/com.farsight.AndroidPinball.javaProject.json]: Planning to revert

**Comment by Omeric** [comment]
Changes look good to me.

Ill update my database with the changes also so future changes keeps this J

**Pull request merged**